src/main/java/com/siawsam/duke/Task.java

```diff
@@ -0,0 +1,44 @@
+package com.siawsam.duke;
+
+import java.io.Serializable;
+
+/**
+ * Represents a serializable task.
+ */
+public class Task implements Serializable {
+    private final String taskDescription;
+    private boolean isDone = false;
+    
+    /**
+     * Constructs a new Task instance containing a task description.
+     *
+     * @param description The task description.
+     */
+    public Task(String description) {
+        taskDescription = description;
+    }
+    
+    /**
+     * Marks the task as "done".
+     *
+     * @return A response indicating the task that is marked as "done".
+     */
+    Response markAsDone() {
+        this.isDone = true;
+        return new Response(Ui.showMarkedAsDone(this));
+    }
+    
+    public boolean includesKeyword(String keyword) {
```

Comment:
Are you missing a javadoc description for this method?

Reply:
I agree, there should be a javadoc here since it is a public method

Reply:
Thanks for catching this guys 👯‍♂️